navigation_onboarding/main.hpp

```diff
@@ -0,0 +1,3 @@
+#include "navigation.hpp"
+
+bool braking_state = false;
```

Comment:
Bad practices:

1. global variable
2. no namespace
3. allocating memory in the header; if two files are to include "main.hpp" independently then they will get different memory locations! in case you're wondering how to do this: put the variable in the .cpp and expose a function that reads/writes it. please don't do this.